api/base/settings/defaults.py

```diff
@@ -228,13 +224,12 @@
     # Uncomment and add "prof" to url params to recieve a profile for that url
     # 'api.base.middleware.ProfileMiddleware',
 
-    # 'django.contrib.sessions.middleware.SessionMiddleware',
+    'django.contrib.sessions.middleware.SessionMiddleware',
```

Comment:
`SystemCheck` doesn't complain on missing this `SessionMiddleware` but our API tests errored with

```
"The Django authentication middleware requires session middleware "
2022-09-02T15:11:33.5352121Z E   AssertionError: The Django authentication middleware requires session middleware to be installed. Edit your MIDDLEWARE setting to insert 'django.contrib.sessions.middleware.SessionMiddleware' before 'django.contrib.auth.middleware.AuthenticationMiddleware'.
```